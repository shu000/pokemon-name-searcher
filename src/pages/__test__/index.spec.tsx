import { useStaticQuery } from 'gatsby';
import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../app';

jest.mock('gatsby');
jest.mock('gatsby-image');

beforeEach(() => {
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  useStaticQuery.mockImplementation(() => ({
    site: {
      siteMetadata: {
        title: `Site Title`,
      },
    },
    placeholderImage: {
      childImageSharp: {
        fluid: undefined,
      },
    },
  }));
});

describe('Index', () => {
  it('snapshot', () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
