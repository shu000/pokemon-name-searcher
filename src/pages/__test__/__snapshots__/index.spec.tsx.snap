// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index snapshot 1`] = `null`;
